pkg/machinery/marker.go

```diff
@@ -28,6 +28,7 @@ var commentsByExt = map[string]string{
 	".go":   "//",
 	".yaml": "#",
 	".yml":  "#",
+	".rs":   "//",
```

Comment:
We cannot add a file that we do not support have officially
Could the extension not be added in your code using the lib?

Reply:
Yeah no problem, I got that:) What I meant: I will try what you suggest above, and actually the way I had implemented it was like what you suggested above. Let me revert to what I had before and compare with your suggestion and see if that works. I dont remember why I avoided that approach, maybe just not to duplicate things (thinking I can have the `rs` extension definition in kubebuilder)

Reply:
@camilamacedo86 ok I've tried and remembered the reason as I tried to use the same approach. Here it is:

![Screenshot 2025-05-25 at 22 04 02](https://github.com/user-attachments/assets/ee9011d6-58aa-40b9-bbe7-eb965a2b106a)

Reply:
So in other words, if we can make the marker struct use public fields instead of private ones, any other plugins can use their own custom markers. 

I can create a PR for this if this sounds good to you?

Reply:
I tried out what I mentioned above here: https://github.com/mabulgu/kubebuilder/commit/4a618554cae2a33c7dcf8a76613d0395d412525a

But getting the following:

![Screenshot 2025-05-26 at 00 27 50](https://github.com/user-attachments/assets/13df6dff-0669-47f3-a664-814194e3b2d9)

So this means we need to extend `NewMarkerFor` as well. I will let you kow once I have a solution for this.

Reply:
Yes, sure. Thank you a lot !!!
Yes, we need to allow people does that and create their own markers for their own extensions.
Please, feel free to raise the PR.